synapse/http/site.py

```diff
@@ -94,13 +216,21 @@ def _started_processing(self, servlet_name):
         )
 
     def _finished_processing(self):
-        try:
-            context = LoggingContext.current_context()
-            usage = context.get_resource_usage()
-        except Exception:
-            usage = ContextResourceUsage()
```

Comment:
Why are we removing the error handling?

Reply:
it seemed pretty redundant. We're already assuming that get_resource_usage doesn't throw elsewhere.